Add documentation for json serialization

HornMichaelS opened the pull request:
<!--**Pull Request Labels**

While not necessary, you can help organize our pull requests by labeling this issue when you open it.  To add a label automatically, simply [x] mark the appropriate box below:

- [ ] feature
- [ ] blocking
- [x] docs

To add a label not listed above, simply place `/label another-label-name` on a line by itself.
-->

I recently found the .jsonObject property very useful for serializing some Apollo iOS data in a format that was compatible with RealmSwift. However, I was only able to find out about the property by digging through some issues on GitHub. So I thought it might be helpful to have it officially documented somewhere. 

If there's a better place for this information, or if it should remain undocumented, please let me know. Thanks for all your work on this!

designatednerd commented:
@HornMichaelS Would you like to fix the conflicts or would you prefer I do the honors?

HornMichaelS commented:
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

HornMichaelS commented:
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

HornMichaelS commented:
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

HornMichaelS commented:
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

HornMichaelS commented:
Alright, should be good to go now! I made a couple further changes, but they're fairly trivial. Changed the syntax of my subheading to markdown, and removed a sentence in the section above mine that was duplicated for some reason

designatednerd commented on docs/source/fetching-queries.md:
````diff
@@ -118,4 +118,24 @@ By default, Apollo constructs queries and sends them to your graphql endpoint us
+  let heroAndFriendsNames = try! HeroAndFriendsNamesQuery.Data(jsonObject: deserialized)
+}
+```
+```
````
looks like an extra set of backticks snuck in here

HornMichaelS replied:
Thanks for catching that! It's fixed now
